include/roboteam_ai/control/controllers/PidController.h

```diff
@@ -0,0 +1,144 @@
+//
+// Created by mrlukasbos on 29-3-19.
+//
+
+#ifndef ROBOTEAM_AI_PID_H
+#define ROBOTEAM_AI_PID_H
+
+#include <tuple>
+#include "BaseController.h"
+
+/**
+ * The basic implementation of a PIDF (proportional - integrative - derivative
+ * - feedback) controller. The actual controller accepts a float reference
+ * and outputs a float controller action. For more info check https://en.wikipedia.org/wiki/PID_controller
```

Comment:
It is now more clear for me, because I know this class is not about the Pelvic Inflammatory Disease. Reference URL to the Wikipedia page which explains more is quite useful.